pkg/ddevapp/config.go

```diff
@@ -1382,6 +1381,26 @@ func WriteImageDockerfile(fullpath string, contents []byte) error {
 	return nil
 }
 
+// IsAllowedProjectLocation returns an error if the project location is not allowed
+func IsAllowedProjectLocation(app *DdevApp) error {
+	homeDir, _ := os.UserHomeDir()
+	if app.AppRoot == filepath.Dir(globalconfig.GetGlobalDdevDir()) || app.AppRoot == homeDir {
```

Comment:
Can we expand this to parents of homedir? I haven't heard of it happening, but what if people accidentally did /home or /Users ?

Reply:
Yes, we can.